x/inflation/genesis.go

```diff
@@ -21,7 +22,10 @@ func InitGenesis(
 
 	// Set genesis state
 	params := data.Params
-	k.SetParams(ctx, params)
+	err := k.SetParams(ctx, params)
+	if err != nil {
+		panic(errors.Wrapf(err, "error setting params"))
```

Comment:
Nit: if above suggestion is committed, need to refactor here as well
```suggestion
		panic(errorsmod.Wrapf(err, "error setting params"))
```